git/git-cheatsheet-de.md

```diff
@@ -35,6 +35,27 @@ git config --global core.eol lf
 
 ## Rebase und Konflikte
 
+### Interaktive Rebase
+
+Rebase ausführen: `git rebase -i HEAD~<Anzahl der Schritte>`
+
+/*
```

Comment:
Hier kannst du auch drei Backticks vor und nach dem Codeblock benutzen, damit das Ganze in Monospace gerendert wird.

Reply:
Habe ich geändert